packages/interfaces/src/index.ts

```diff
@@ -60,6 +60,12 @@ export interface TimeFilter {
   endTime: Date;
 }
 
+export type Cursor = {
+  digest?: Uint8Array;
+  senderTime?: bigint;
+  pubsubTopic?: string;
```

Comment:
non-blocking: why are these optional?

Might actually be answered as part of https://github.com/waku-org/js-waku/pull/1024#discussion_r1029167030.

Reply:
merging this PR for now; can address the change in the next PR as @LNSD clarifies.